main.go

```diff
@@ -317,7 +317,7 @@ func getTracer(opts Options, out output) (opentracing.Tracer, io.Closer) {
 		tracer opentracing.Tracer = opentracing.NoopTracer{}
 		closer io.Closer
 	)
-	if opts.TOpts.Jaeger {
+	if opts.TOpts.Jaeger && !opts.TOpts.NoJaeger {
```

Comment:
Consider override order. Flags on the CLI need to supersede flags from the config file. To make this work, we’ll have to push the mush heavier boulder up the hill—read the options once from config, once from flags, and merge them manually. The same applies for headers and baggage.